tests/integration-tests/tests/schedulers/test_awsbatch.py

```diff
@@ -45,6 +45,27 @@ def test_awsbatch(pcluster_config_reader, clusters_factory, test_datadir, caplog
     _test_job_kill(remote_command_executor, timeout)
 
 
+@pytest.mark.batch_dockerfile_deps
+@pytest.mark.usefixtures("region", "os", "instance", "scheduler")
+def test_awsbatch_defaults(pcluster_config_reader, clusters_factory, test_datadir, caplog, region):
```

Comment:
If I'm not wrong the entire code is the same of the test_awsbatch one or it can be put in common. The only change we're doing is in the configuration file but test code seems minimal.